pkg/remotewrite/trend.go

```diff
@@ -11,6 +11,7 @@ import (
 	"go.k6.io/k6/metrics"
 )
 
+// TrendStatsResolver is a map of trend stats name and their relative resolver function
```

Comment:
Autospot as a reminder, the comment is nice but I guess it is not required to export it